build.sbt

```diff
@@ -349,7 +364,7 @@ lazy val uiSettings = Seq(
 lazy val dockerBuildxSettings = Seq(
   createBuildx := {
     streams.value.log("Creating docker buildx instance")
-    haltOnCmdResultError(Process("docker buildx create --use --name multi-arch-builder", baseDirectory.value).!)
+    Process("docker buildx create --use --name multi-arch-builder", baseDirectory.value).!
```

Comment:
we don't want to halt? :)

Reply:
Well, that command will return either success with new Buildx instance or just info about existing instance.

Reply:
So if the instance exist, the command ends in error?